packages/core/src/amazonqFeatureDev/constants.ts

```diff
@@ -14,6 +14,9 @@ export const featureDevChat = 'featureDevChat'
 
 export const featureName = 'Amazon Q feature development'
 
+// Max allowed size for file collection
+export const maxRepoSize = 200 * 1024 * 1024
```

Comment:
nit: please indicate the units used.

e.g. `maxRepoSizeBytes`
Who knows what you mean, unicode chars weigh more than 1 byte